README.md

```diff
@@ -1,54 +1,45 @@
-# Docker Proxy for ghcr.io
+# Proxy servers for OpenSAFELY
 
-This hosts the Dockerfile for building an nginx docker image that will proxy
-traffic to ghcr.io for the OpenSAFELY project.
+To secure and limit access to external services, the OpenSAFELY platform
+maintains a proxy service. OpenSAFELY backends explicitly use these proxies
+when they need to access external data.
 
-OpenSAFELY publishes its open docker images via ghcr.io, which are needed by
-backends to run OpenSAFELY studies. However, we don't want to require backends
-to allow network access to ghcr.io, as that could in theory be used to
-exfiltrate data.
+This repository produces a Docker image that uses nginx to host two proxy
+domains:
+ 
+ * github-proxy.opensafely.org: this provides access to *only* opensafely
+   repositories hosted on https://github.com, and not other repositories.
 
-So this proxy provides a single point of read-only access to only the
-OpenSAFELY images on ghcr.io, by only proxying GET requests to /v2/opensafely/
-urls.
+ * docker-proxy.opensafely.org: this provides read only access to docker images
+   published by specific organistions on https://ghcr.io, the Github Container
```

Comment:
nit: I think `specific Github organisations` would be better here (which means some of the several GH orgs our team has), `specific organisations` implies to me real-world orgs (like, Canonical, DL, ...)

Reply:
good call, will change